test/cluster/network_policy_test.go

```diff
@@ -674,6 +674,16 @@ func doTestNetworkPolicy(
 				enableTLS,
 			)
 
+			if !multiPodPerHost {
+				if enableTLS {
+					aeroCluster.Spec.AerospikeConfig.Value["network"].(map[string]interface {
+					})["service"].(map[string]interface{})["tls-port"] = serviceTLSPort + GinkgoParallelProcess()*10
+				}
+
+				aeroCluster.Spec.AerospikeConfig.Value["network"].(map[string]interface {
+				})["service"].(map[string]interface{})["port"] = serviceNonTLSPort + GinkgoParallelProcess()*10
+			}
+
```

Comment:
As this is repeated multiple time, see if we can move this to a common something like `randomizeServicePorts`